runelite-client/src/main/java/net/runelite/client/plugins/woodcutting/config/ClueNestTier.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2021, Tal <https://github.com/talsk>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.woodcutting.config;
+
+public enum ClueNestTier
+{
+	BEGINNER(0),
+	EASY(1),
+	MEDIUM(2),
+	HARD(3),
+	ELITE(4),
+	DISABLED(999);
+
+	private final int tier;
```

Comment:
This can be replaced by using `ClueNestTier.ordinal()` comparisons. I recommend using the nest item IDs instead so the tier can be fetched from the item spawn subscriber.

Reply:
I moved to use ordinal, kinda forgot about it, thanks! As per the previous comment I moved the map to the enum so it absolves the need to use the nest item IDs, or any specific value whatsoever.